specs/CameraCaptureUI/CameraCaptureUI.md

```diff
@@ -0,0 +1,387 @@
+| MaxResolution | Determines the maximum resolution the user will be able to select. | CameraCaptureUIMaxPhotoResolution |
+
+
+
+
+## CameraCaptureUIVideoCaptureSettings Class
+Provides settings for capturing videos. 
+The settings include format, maximum resolution, maximum duration, and whether or not to allow trimming.
+
+## CameraCaptureUIVideoCaptureSettings Properties
+| Name | Description | Value |
+|-|-|-|
+| AllowTrimming | Determines whether the user can trim the captured video.| Boolean |
+| Format | Determines the format that captured videos will be stored in.| CameraCaptureUIVideoFormat |
+| MaxDurationInSeconds | The maximum duration of the video that the user can capture.| Single |
+| MaxResolution | Determines the maximum resolution the user will be able to select.| CameraCaptureUIMaxVideoResolution |
+
+
+## CameraCaptureUIMode enum
+
+Determines whether the user interface for capturing from the attached camera allows capture of photos, 
+videos, or both photos and videos.
+
+| Name | Value | Description |
+|-|-|-|
+| PhotoOrVideo | 0 | The user interface allows capture of photos and videos. |
+| Photo | 1 | The user interface allows capture of photos only.|
+| Video | 2 | The user interface allows capture of videos only. |
+
+
+## CameraCaptureUIPhotoFormat enum
+Determines the format for photos captured with CameraCaptureUI.
+
+| Name | Value | Description |
+|-|-|-|
+| Jpeg | 0 |JPEG format. |
+| Png | 1 | PNG format. |
+| JpegXR | 2 | JPEG-XR format. |
+
+
+## CameraCaptureUIVideoFormat enum
+Determines the format for saving captured videos.
+
+| Name | Value | Description |
+|-|-|-|
+| Mp4 | 0 |MP4/H.264/AAC format.|
+| Wmv | 1 | WMV format. |
+
+## CameraCaptureUIMaxPhotoResolution enum
+Determines the highest resolution the user can select for capturing photos.
+
+| Name | Value | Description |
+|-|-|-|
+| HighestAvailable | 0 | The user can select any resolution. |
+| VerySmallQvga | 1 |The user can select resolutions up to 320 X 240, or a similar 16:9 resolution. |
+| SmallVga | 2 | The user can select resolutions up to 320 X 240, or a similar 16:9 resolution.|
+| MediumXga | 3 | The user can select resolutions up to 1024 X 768, or a similar 16:9 resolution. |
+| Large3M | 4 | The user can select resolutions up to 1920 X 1080, or a similar 4:3 resolution. |
+| VeryLarge5M | 5 | The user can select resolutions up to 5 MP.|
+
+
+## CameraCaptureUIMaxVideoResolution enum
+Determines the highest resolution the user can select for capturing video.
+
+| Name | Value | Description |
+|-|-|-|
+| HighestAvailable | 0 | The user can select any resolution. |
+| LowDefinition | 1 | The user can select resolutions up to low definition resolutions.|
+| StandardDefinition | 2 | The user can select resolutions up to standard definition resolutions.|
+| HighDefinition | 3 | The user can select resolutions up to high definition resolutions. |
```

Comment:
Should these be defined in more specifics?  I assume SD means "480p" and HD is "720p" or possibly higher.  But that's a guess.

Reply:
I agree the values do seem vague, but it is how the historic documentation defines the values. We can talk with the Camera team owners if they would like to take up this suggestion.

Reply:
It might have been intentionally vague in the existing API. We've long had trouble deciding between flexibility like this vs having to make up names like xx-small. I don't think we should change the API from existing here, but if we can be more detailed in the doc like we are with MaxPhotoResolution that would be fine.

Reply:
The values listed in the documentation were carried over directly from the existing documentation without any changes. We can reach out to the Camera team to discuss if they’d like to consider refining these definitions.